modules/instrument_builder/jsx/react.questions.js

```diff
@@ -426,6 +419,13 @@ class DateOptions extends Component {
   render() {
     let minYear = this.props.element.Options.MinDate;
     let maxYear = this.props.element.Options.MaxDate;
+    let dateFormat = this.props.element.Options.dateFormat;
+    if (minYear && minYear.length > 4) {
+      minYear = minYear.substr(0, 4);
```

Comment:
the changes here to minYear & maxYear made it so the fields would show the correct years when editing the Date field.